src/main/msp/msp.c

```diff
@@ -3280,15 +3281,39 @@ static mspResult_e mspCommonProcessInCommand(mspDescriptor_t srcDesc, uint8_t cm
         break;
 
     case MSP_OSD_CHAR_WRITE:
-#ifdef USE_MAX7456
+#if defined(USE_OSD)
         {
-            uint8_t font_data[64];
-            const uint8_t addr = sbufReadU8(src);
-            for (int i = 0; i < 54; i++) {
-                font_data[i] = sbufReadU8(src);
+            osdCharacter_t chr;
```

Comment:
Has this been tested and confirmed to work with a MAX7456?

Reply:
The MAX7456 driver continues to read only 54 bytes per character, it's not affected by this change.